source/crud/tests/v2/unacknowledged-deleteOne-hint.yml

```diff
@@ -0,0 +1,57 @@
+runOn:
+  - { minServerVersion: "4.3.4" }
+
+data:
+  - {_id: 1, x: 11}
+  - {_id: 2, x: 22}
+
+collection_name: &collection_name 'DeleteOne_hint'
+
+tests:
+  -
+    description: "Unacknowledged deleteOne with hint string"
+    operations:
+      -
+        object: collection
+        collectionOptions: &collection_options
+          writeConcern: { w: 0 }
+        name: deleteOne
+        arguments:
+          filter: &filter { _id: 1 }
+          hint: "_id_"
+        result: &result
+          deletedCount: 0 # Delete is unacknowledged
+    expectations:
+      -
+        command_started_event:
+          command:
+            delete: *collection_name
+            deletes:
+              -
+                q: *filter
+                hint: "_id_"
```

Comment:
Note to self: Add `limit` fields to all command started events for delete commands.

Reply:
New note to self: done.